airflow/utils/net.py

```diff
@@ -0,0 +1,16 @@
+from airflow.configuration import conf, AirflowConfigException
+import socket
+
+
+def get_hostname(default=socket.getfqdn):
+    """
+    A replacement for `socket.getfqdn` that allows configuration to override it's value.
+
+    :param callable|str default: Default if config does not specify. If a callable is given it will be called.
+    """
+    try:
+        # If this is called `get`, why does it not match python semantics?
```

Comment:
Does this comment really belong here?

Reply:
Would you like me to remove it? I will happily :)

Reply:
Removed